### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by riskmaris**
# Setup and mobile-first
Added 2 different background images for the contact form section.
Added style for the title of the contact form
Added style for input fields

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @riskmaris, @hagan23, and @rahimialijan, 

roja

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional README file. Please 🙏🏽 add the information of all members in the Authors section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -271,19 +270,26 @@ <h3 class="connect">LET'S CONNECT</h3>
...
-                  
+              <!-------------------------------------- Contact form------------------------------->
+            <div class="about-contact">
+              <div class="title-container">
+                <h2 class="contact">Contact Me</h2>
+                <p class="para">If you have an application you are interested in<br>developing, 
+                  a feature that you need built or a project<br>that needs coding. i'd love to help with it</p>
+              </div>
+              <div class="form-container">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="labels">
```
Comment (Shedrack-Sunday) [inline index.html:281]: - [ ] You need to add the `id` to the formspree to make it submit successfully. use this [Link](https://formspree.io/html/) to see the process of how to do so 👍 
- Also capitalize the `POST` method to make it a standard highlight understandable to all 👍

File: index.html [hunk 2]
```diff
@@ -271,19 +270,26 @@ <h3 class="connect">LET'S CONNECT</h3>
...
+                    <input type="text" name="name" id="name" placeholder="enter name" required>
+                    <input type="email" name="email" id="email" placeholder="enter email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" placeholder="message" required></textarea>
```
Comment (Shedrack-Sunday) [inline index.html:284]: - [ ] Name input should have a `maxlength="30"` attribute.
- [ ] The  Message input should have a  `maxlength="500"` attribute.
- This is right to give proper validations 👍

File: index.html (hunk 1 above)
Comment (Shedrack-Sunday) [inline index.html:281]: - [ ] The padding on the forms needs a little bit of adjustment. It does not give a nice UI to have text starting on the margin attached.

## View 👍 
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @riskmaris           👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
✔️ Good PR summary
✔️ Nice code implementation
  
# Required Changes ♻️

- [x] Your form is not submitting properly, because it lacks an `ID`. I've given a detailed explanation of this below 👍 
- There's a bit of styling that needs to be an adjustment 


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @riskmaris , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
soclose


 ## Highlights
- [ ] The form spree service has been used correctly :+1: 
- [ ] Descriptive PR `title` and `summary` :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @Team :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Textarea has max-length :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Professional ReadMe documentation :heavy_check_mark: 

## Optional suggestions

_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**